.travis.yml

```diff
@@ -16,6 +16,7 @@ env:
 
 before_install:
   - bundle install
+  - gem install cocoapods --pre
```

Comment:
Install the 1.6.0 beta because of an issue with pod lib lint 
I don't think it should be merged, but otherwise, we will have to wait for the 1.6.0 official release :))
@omaralbeik @SD10 @guykogus

Reply:
I think it's fine, however this should be marked in the `Gemfile` with
`gem 'cocoapods', '>= 1.6.0.beta.1'`
rather than forcing the install here. In that case you should run `bundle update` locally and add `Gemfile.lock` to git, cos it looks like we don't have it currently 🤔.

If you're not keen to set the pre-release version and all that, I wouldn't be against having a `swift-4.2` branch and we can merge into there (I also have some new extensions for it that I'd like to add 😁). Then once cocoapods does an official release for 1.6.0 we can merge into `master`.

Reply:
I'm a little bit skeptical about putting this on gem file because it is just a temporary workaround, when they release it we probably will remove it because it's gonna be already installed on Travis machine by default. That's why I just put the script :))